### DISCUSSION TITLE
IWP 결제 오류-결제 정보 누락.

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#558] [Issue Report (Legacy)]
현재 서비스중인 게임에서 결제가 누락된다는 제보를 듣고, 오류나는 테스트영상을 올립니다.

https://cafe.naver.com/zepetostudio/16

https://youtu.be/RUD2eshOsy8

영상을 보면, 첫번째 플레이시 잼 2개짜리 아이템을 결제시도합니다.
테스트 아이디라 실제 결제가 이루어지지 않는다는 안내와 함께 결제가 완료되었다는 제페토앱 팝업이 보입니다.
그뒤 결제된 골드가 지급되는 게임팝업이 떠야하지만 아무런 반응이 없습니다.
(결제에 오류가 생기면 오류메시지가떠야합니다.)

​2개짜리 아이템 이외의 다른아이템은 결제가 끝나고 골드 지급팝업이 정상적으로 확인이됩니다.

여기까지면, 첫번째 아이템에관련된 코드가 문제가 있을수있습니다.(아이템아이디나, 콜백연결등등...)

이후 게임을 나갔다 다시 들어온 영상에서(동일한 버전확인을 위해 원테이크로 찍었습니다.) 동일하게 2개짜리 아이템을 구매하면

2개짜리도 정상적으로 골드 지급이되는게 확인이됩니다.
(아이템아이디나, 다른 기능구현에 문제가 없었다는 뜻이됩니다.)

​영상으로 찍진 못했지만, 게임을 나갔다 오지 않아도 정상작동으로 되는경우도 있는것으로 보입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 해당 현상 확인해 보겠습니다. (첨부해주신 카페글은 열람이 불가하네요🥲)

**Comment by Andwhy81kr** [comment]
해당이슈에 대해 좀 더 정확한 재현 스탭과, 회피방법을 찾았네요.

버전이 올라가면 수정될것으로 보이지만 버전업하기 껄끄러운분들은 참고해봐주세요.

https://cafe.naver.com/zepetostudio/30
